libs/core/src/lib/services/usb.serial.service.ts

```diff
@@ -8,10 +8,17 @@ export class UsbSerialService {
     protected static instance;
 
     /**
-     * Get instance
+     * @Deprecated - Get instance,  Use the new {@link createInstance} instead.
```

Comment:
The documentation generated is a little weird, there is a bullet point in the documentation. Could you change this, please ? 
I think we should have something like : 
/**
  * Get instance, get the instance of the usb serial service.
  * @returns an instance of the usb serial service
  * @deprecated Use the new {@link createInstance} instead.
  */